utils/runtime_test.go

```diff
@@ -0,0 +1,61 @@
+package utils
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	sharedpb "github.com/tron-us/go-btfs-common/protos/shared"
+	"github.com/tron-us/go-common/constant"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCheckRuntimeDBRD(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+	shared := new(sharedpb.SignedRuntimeInfoRequest)
+	os.Setenv("SS_DB_URL", "postgres://uchenna:Q@vl321!@localhost:5432/db_status")
+	os.Setenv("SS_RD_URL", "redis://uchenna:@127.0.0.1:6379/4?pool=25&process=2")
```

Comment:
please pass in such ENV outside of tests(should be set in makefile, or manually for now), and test ENV should be different from the local running ones
e.g SS_DB_URL vs SS_DB_TEST_URL

Reply:
ok